packages/maskbook/src/plugins/Wallet/types/portfolio.ts

```diff
@@ -25,6 +31,7 @@ export interface Transaction {
     failed: boolean
     pairs: TransactionPair[]
     gasFee: TransactionGasFee | undefined
+    transactionType: string
```

Comment:
Shouldn't it be `FilterTransactionType`?

Reply:
It needs to be compatible with types from different data sources